packages/jupyter-ai/jupyter_ai/chat_handlers/fix.py

```diff
@@ -96,16 +113,35 @@ async def process_message(self, message: HumanChatMessage):
         extra_instructions = message.prompt[4:].strip() or "None."
 
         self.get_llm_chain()
-        with self.pending("Analyzing error", message):
```

Comment:
Probably would be good to adjust API so that custom pending message can be passed to `stream_reply` :)

Reply:
@krassowski TY for the suggestion. I made the changes though I am not sure it is worth updating `stream_reply` in `base.py` for a custom pending message with `/fix`. Changing the message from "Generating response" to "Analyzing error" when the message flashes for a brief second in the chat window may not enhance UX enough to make adding a new parameter to the `stream_reply` function worthwhile. But, I can also see other settings in which we may want to pass custom pending messages. It is a trade off.